### PR TITLE
[Feature] wait - chart timeout keyword

### PR DESCRIPTION
**Pull request by theyer**
**What is the purpose of this pull request?** Use tiller's native timeout feature

**What issue does this pull request address?**: ARMADA-42

**Notes for reviewers to consider**:

**Specific reviewers for pull request**:
@gardlt @drewwalters96

### CODE REVIEW

File: armada/handlers/tiller.py [hunk 1]
```diff
@@ -23,7 +23,7 @@
 from ..utils.release import release_prefix
 
 TILLER_PORT = 44134
-TILLER_VERSION = b'2.1.3'
+TILLER_VERSION = b'2.1.3' # need to be updated?
```
Comment (theyer) [inline armada/handlers/tiller.py]: @gardlt

File: armada/handlers/tiller.py [hunk 2]
```diff
@@ -23,7 +23,7 @@
 from ..utils.release import release_prefix
 
 TILLER_PORT = 44134
-TILLER_VERSION = b'2.1.3'
+TILLER_VERSION = b'2.1.3'  # need to be updated?
```
Comment (gardlt) [inline armada/handlers/tiller.py]: This does need to be updated, Good Catch!

File: armada/handlers/armada.py [hunk 3]
```diff
@@ -116,6 +113,18 @@ def sync(self):
             if chart.release_name is None:
                 continue
 
+            # retrieve appropriate timeout value if 'wait' is specified
```
Comment (gardlt) [inline armada/handlers/armada.py]: we should be able to break this down. a little. 
why are you converting self.timeout to int?
getattr(chart, "var", None)

Comment (theyer) [inline armada/handlers/armada.py]: done

File: armada/handlers/tiller.py (hunk 2 above)
Comment (theyer) [inline armada/handlers/tiller.py]: done

**Pull request merged**